intervals/interval.py

```diff
@@ -321,7 +324,7 @@ def length(self):
     def radius(self):
         if self.length == inf:
             return inf
-        return float(self.length) / 2
+        return self.length / 2
```

Comment:
That appears to be an awkward part of the commit, that perhaps should have gone elsewhere, since the intention was to only move six. Let me try to change that PR to remove this change (or easier, create a new PR). 
I think my intention was to actually have `from __future__ import division`, after which the float conversion is not necessary anymore (not for the resulting value, but it is still for the resulting type)
.